server/SeparateStainsMacenkoPCA/SeparateStainsMacenkoPCA.xml

```diff
@@ -30,7 +30,6 @@
       <longflag>samplePercent</longflag>
       <description>Percentage of pixels to sample.  Specify either this or --sampleApproximateTotal</description>
       <constraints>
-	<minimum>0</minimum>
 	<maximum>1</maximum>
```

Comment:
It is used as a fraction in `histomicstk.utils.sample_pixels`, despite the name (which is `sample_percent` there as well). I'm happy to change all related names to the appropriate variety of `sample_fraction`.